src/semver.ts

```diff
@@ -65,6 +66,17 @@ export class SemVer {
     this.prefix = prefix;
   }
 
+  static copy(semver): SemVer {
+    return new SemVer({
```

Comment:
cannot you use `new SemVer({...semver})`?

Reply:
No, because of the private `_build` in there.
I suppose I could do:
```
    return new SemVer({
      build: semver.build,
      ...semver,
    });
```